run_sweep.py

```diff
@@ -102,7 +102,7 @@ def _run_model_test(model_path: pathlib.Path, test: str, device: str, jit: bool,
         result.results["latency_ms"] = run_one_step(func, device)
         # if the model provides eager eval result, save it for cosine similarity
         correctness = task.get_model_attribute(correctness_name)
-        if correctness:
+        if not correctness == None:
```

Comment:
`correctness` value could be `0.0`

Reply:
> `if correctness is not None:` to be more pythonic

It looks like `is not` syntax is discouraged in Python 3.8 release note: https://docs.python.org/3.8/whatsnew/3.8.html#changes-in-python-behavior

Reply:
This is for comparisons with strings/ints (that indeed throw warning), `is not None` is still a legit construct, no warning.

Reply:
> This is for comparisons with strings/ints (that indeed throw warning), `is not None` is still a legit construct, no warning.

Thanks for pointing out! Mark this as resolved.